examples/deployment/README.md

```diff
@@ -51,7 +51,9 @@ a cloud.
 
 ### Run in GCP
 
-TODO
+There is a good
```

Comment:
While you're here, maybe worth adding a link to [the kubernetes docs](https://github.com/google/trillian/tree/master/examples/deployment/kubernetes) too?

Reply:
Done. Also moved the AWS instructions into the "aws" directory, for consistency and to avoid people missing them when linked directly to the "aws" directory.